webapp/packages/plugin-connections/src/ConnectionAuthService.ts

```diff
@@ -86,6 +104,7 @@ export class ConnectionAuthService extends Dependency {
         connection: key,
         networkHandlers,
         resetCredentials,
+        distributed: this.serverConfigResource.distributed,
```

Comment:
seems like it's better to get this data in dialog instead (cuz it's not a parameter but app env)